datasets/covtype.py

```diff
@@ -11,9 +12,18 @@ class Dataset(BaseDataset):
     install_cmd = 'conda'
     requirements = ['pip:scikit-learn']
 
+    parameters = {
+        'standardized': [False, True]
+    }
+
     def get_data(self):
         X, y = fetch_covtype(return_X_y=True)
         y[y != 2] = -1
         y[y == 2] = 1  # try to separate class 2 from the other 6 classes.
+
+        if self.standardized:
+            scaler = StandardScaler()
+            X = scaler.fit_transform(X)
```

Comment:
I would simplify and always standardize.

Reply:
Ok -- only for Covtype? It would make sense to do it for all non-sparse datasets, right?